mir-ci/readme.md

```diff
@@ -1,9 +1,9 @@
 # Ubuntu Frame Integration Tests
```

Comment:
Should we move this readme to the top level directory? Or maybe add a readme at the top level that references this?